src/components.py

```diff
@@ -15,29 +15,10 @@
 
 
 def render_app_header():
-    app_info = get_application_info()
-    app_name = I18N_APP_NAME
-    app_description = I18N_APP_DESCRIPTION
-    external_access_enabled = app_info.get('externalAccessEnabled', False)
-    app_url = app_info.get('applicationUrl', None)
-
     st.logo(APP_LOGO)
 
-    # Remove flex grow so header does not take half the app height
-    with sal.columns('no-flex-grow'):
-        col0, col1 = st.columns([0.8, 0.2], vertical_alignment="center")
-        with col0:
-            with sal.subheader("app-header", container=col0):
-                col0.subheader(app_name, anchor=False)
-            if app_description:
-                col0.caption(app_description)
-
-        # Style this column to move the button to the right side
-        with sal.column('justify-end', 'flex-row', container=col1):
-            if external_access_enabled and app_url:
-                with sal.button('share-button', container=col1):
-                    if col1.button(I18N_SHARE_BUTTON, key='share-button'):
-                        show_share_dialog(app_url)
+    with sal.image('centered-image'):
+        st.image(APP_LOGO)  # Set path of the company image to show centered above the chat messages
```

Comment:
@shu this renders the app logo in the center of a div or streamlit part. I just copied this into the part where the diaglogue modal appears